### PR TITLE
Ghseeli/real week 1

### PR DESCRIPTION
**Pull request by ghseeli**
Here are the updated real analysis notes up to the end of this week. Of need of special review is probably the most recent lecture because I did not understand all of it, especially the proof of the last big theorem. In keeping with the review schedule, I have added @MareoRaft as a reviewer, but @cjl8zf , please feel free to also look at this, especially the specific section mentioned above.

### CODE REVIEW

**Comment by MareoRaft** [comment]
so if i want to actually make changes to this, i should accept the pull request and then commit on top of it?

i've never used this feature before.

**Comment by ghseeli** [comment]
So, you have a few options. If you want to directly make changes, just check out the branch `ghseeli/real-week-1` and commit on it. Alternatively, you can just make comments on the changes about what needs to be changed and I will do it. Don't merge the pull request, though. Once you have done your review, just give a positive review and then I will look at your changes and if I agree with them, I will merge the PR.

**Comment by ghseeli** [comment]
Check this out: https://help.github.com/articles/about-pull-request-reviews/

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: he meant "measurable space"

Comment (MareoRaft) [reply Real Analysis/real-analysis-notes.tex]: An element of A is a "measurable set"

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: again "spaces"

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: Maybe "projection" maps is a better description than "natural".

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: I think it would improve the readability of this paragraph if we just picked $X$ or $A$ and used it throughout, instead of using both.

Comment (MareoRaft) [reply Real Analysis/real-analysis-notes.tex]: Also, can we take the product across A if A is not countable?  I'm wondering how general this product really can be.  I was thinking of (y_alpha) as a sequence, but then realized it may not be.

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: This is a definition

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex:161]: Suggest "The *product sigma algebra* on X is the smallest sigma algebra generated by this data.  Namely, ..."

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex:165]: We can probably just cut out these four lines

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: "\in" instead of "\subset"

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: It should the the SET of products, where the family of E's can change each time.  That is, it should be \Ep_2 here, not a single element of \Ep_2.

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: suggest reversing order of statements around "iff" for readability

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: Did you mean the product sigma algebra?  We haven't defined a sum sigma algebra, unless it's being defined here or something...

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex:229]: nix the "for all alpha".  instead, it should be "for all families (E_alpha) where ever E_alpha in \Ep_\alpha"

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex:256]: This made me laugh.

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: "subset", not "in"

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: This is really a definition of "canonically defined topology" and "equivalent", although the latter is an ambiguous name.

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex:377]: I think it's meant to say that EVERY distance is less than r, not an increasing list of things.

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [inline Real Analysis/real-analysis-notes.tex]: and so the last thing should be the ball B(x_{i_k},r) or radius r, like the first thing.

**Review by MareoRaft — CHANGES_REQUESTED** [review]
All of the comments are meant to be addressed.  I also committed some "minor changes" ontop of your latest version.  These are so minor that you don't need to look at them.  So just the 19 comments, even the ones that for some reason say "outdated".

File: Real Analysis/real-analysis-notes.tex
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex]: I agree we should be consistent and use just $X$ after the initial sentence. An uncountable product looks like a function.

File: Real Analysis/real-analysis-notes.tex
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex:165]: I disagree. The previous part is a definition that specifies notation, the latter part is a very condensed mathematical expression. I can rework it to make the first part more related to definition 1.12.

**Comment by ghseeli** [comment]
Alright, thanks for the comments! I will commit changes later tonight.

File: Real Analysis/real-analysis-notes.tex
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex:377]: Okay, that makes more sense!

File: Real Analysis/real-analysis-notes.tex
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex]: Haha yes, I did!

File: Real Analysis/real-analysis-notes.tex
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex:229]: Hmmm, this is the exact notation he used in class and I think they are equivalent. I would want to err on the side of what he put on the board when it is not incorrect.

File: Real Analysis/real-analysis-notes.tex
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex]: I don't understand the remark about "equivalent." A topology is just a subset of the power set of X. If two topologies are equivalent if they are equivalent as sets.

**Comment by ghseeli** [comment]
Alright, @MareoRaft , I think I have addressed all your comments. Please give it a look when you have time and update your review as you see fit. Once you give a positive review, I will do merge.

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [reply Real Analysis/real-analysis-notes.tex:229]: I don't think "for all alpha" is even sensical.  Haven't we taken the product over all alpha?  So the resulting product is no longer indexed by alpha.  So there is simply no alpha to refer to.

File: Real Analysis/real-analysis-notes.tex
Comment (MareoRaft) [reply Real Analysis/real-analysis-notes.tex]: Prerequisite:  Given an X, a distance d can be used to generate a topology on X.  We use the distance to define sets of the form B(p, r).  We then use those balls as a basis for the topology.

definition of equivalent:
He's saying that given a fixed X, two distances d1 and d2 are called "equivalent" if they generate exactly the same topology.  

Does that clarify?

**Review by MareoRaft — APPROVED** [review]
I found one last error:  In prop 1.18 (a), on the LHS, it should be the PRODUCT of the borel sets of the Xj.

I haven't looked at your changes specifically, but I'm assuming you followed my suggestions except for the one place where you disagreed.  I trust you so as long as it makes sense to you, i'm happy.

Thanks for your quality work.

File: Real Analysis/real-analysis-notes.tex
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex]: Yeah, that clarifies. I think lines 282--283 reading: 
```
If $d_1,d_2$ are two metrics on $X$, we say that $d_1$ and $d_2$ are
equivalent (topologically) if and only if $\Top(d_1) = \Top(d_2)$.
```
Are rigorous enough to clarify that confusion in the notes?

File: Real Analysis/real-analysis-notes.tex
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex:229]: Hmmm. I read that as saying "\F_2 is the set of elements of the form \prod_{\alpha \in A} E_\alpha where we restrict the E_\alpha's such that, for all \alpha \in A, E_\alpha is in \Ep_\alpha." I do agree that perhaps the notation is a little off-putting. Personally, when written in a sentence, I would prefer to say "\F_2 is the set of elements of the form \prod_{\alpha \in A} E_\alpha where we restrict the E_\alpha's such that E_\alpha is in \Ep_\alpha for all \alpha in A," but I guess he chose that ordering because the E_\alpha is dependent on your choice of \alpha, so the choice of \alpha is logically prior?

**Comment by ghseeli** [comment]
@MareoRaft , Thank you for all your comments. You definitely did a thorough job catching (important!) mathematical mistakes and, even when we disagree, forced me to think more about and engage with the material. I think this has been a very productive exercise!

**Pull request merged**
